entgql/annotation.go

```diff
@@ -123,22 +123,46 @@ func OrderField(name string) Annotation {
 	return Annotation{OrderField: name}
 }
 
-// Bind returns a binding annotation.
+//Bind returns a binding annotation.
+// entgql.Bind() main purpose's in the Ent schema definition is to improve the efficiency of queries using GraphQL.
+// The Bind() annotation is enabled by default, use Unbind() to opt-out.
 //
-// No-op function to avoid breaking the existing schema.
-// You can safely remove this function from your scheme.
-//
-// Deprecated: the Bind option predates the Unbind option, and it is planned
```

Comment:
@dorlib hello, this annotation was planned to remove. So, why did you remove our deprecated note?

Reply:
The purpose was to extand the docs about this option untill then, and also in order to understand Unbind better.
I think i've added this note before the example and i would like to hear improvement suggestions to this commit 🙂

Reply:
Please keep the keyword `Deprecated:` as is. To allow go linter mask the function is deprecated in their code.